// [[Reason: #56992 Type printer preserves binding element aliases.]] ////

//// [tests/cases/compiler/declarationEmitBindingPatterns.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,8 +1,8 @@
 
 
 //// [declarationEmitBindingPatterns.d.ts]
-declare const k: ({ x: z }: {
+declare const k: ({ x }: {
     x?: string;
 }) => void;
 declare var a: any;
 declare function f({}?: any, []?: any, { p: {} }?: any): void;
